src/utils/horizon.ts

```diff
@@ -54,7 +54,7 @@ export function getAssetReserve(
       stellar_reserve_amount += Number(balanceLine?.selling_liabilities);
     }
     if (asset.isNative()) {
-      stellar_reserve_amount += 1.5 + 0.5 * account.subentry_count; // add 1 XLM for gas headroom
+      stellar_reserve_amount += 10.5 + 0.5 * account.subentry_count; // add 10.5 XLM for gas headroom
```

Comment:
Can we change this to `11 + 0.5 * account.subentry_count`?

By default, the base reserve for an account is 1 XLM, and 0.5 per subentry, which was the actual source of the bug.

with 11, we get 10 XLM left for gas headroom.